wp-content/themes/orient-theme/homepages/skyscraper.php

```diff
@@ -49,6 +49,28 @@
 		</div>
 	</div>
 
+	<div class="row">
+		<div class="col col-sm-12 hidden-lg hidden-md">
+			<?php echo home_render("M6", 16); ?>
```

Comment:
I wonder if we should use a for loop instead of writing all the function calls out like this? See below for an example earlier in the file:

https://github.com/BowdoinOrient/bowpress/blob/7fbdc73362e3cb6bfdb365066808b5431037cab1/wp-content/themes/orient-theme/homepages/skyscraper.php#L46-L48